src/analysis/utils/runtime-utils.ts

```diff
@@ -73,16 +73,17 @@ export type NormalizedComputeRegion = NominalType<string, 'ComputeRegion'>;
 
 // TODO: test when zone and region have types
 export const getNormalizedComputeRegion = (config: GoogleRuntimeConfig | AzureConfig): NormalizedComputeRegion => {
+  const regionNotFoundPlaceholder = 'Unknown';
```

Comment:
sorry for the trail of comments, trying to keep up - I'm definitely missing something, because `region` looks like a required field for all of these `*Config* types. How is a null value getting into this method? Or are we guarding against the empty String case?

Reply:
It is optional here: 
<img width="1126" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/6465084/dc78235d-bee3-4170-b1a9-f944c35416c1">

Reply:
Should probably be added to the UI model...

Reply:
Is the UI type just not enforced then? Shouldn't Typescript be complaining? Or does it not bc some of the pipeline is still Javascript?

Reply:
If we do expect a non-null value, we should normalize it in the data layer, not in the utils logic, right?

Reply:
Resolved in person, it is now normalized in the data layer